extension.js

```diff
@@ -5,17 +5,28 @@ const path = require('path');
 exports.activate = function(context) {
 	// The server is implemented in java
 	let serverModule = context.asAbsolutePath(path.join('server-build', 'langserver.jar'));
-    
+	const main = 'org.ballerinalang.langserver.launchers.stdio.Main';
+
+	const config = workspace.getConfiguration('ballerina');
+
+	// in windows class path seperated by ';'
+	const sep = process.platform === 'win32' ? ';' : ':';
+	
+	if (config.sdk) {
+		// sdk path is set in configurations
+		serverModule = context.asAbsolutePath(path.join('server-build', 'langserver-no-bal-deps.jar'));
```

Comment:
shall we rename **langserver-no-bal-deps.jar** into something else? Since VSCode do not use the alternative version of the lang server jar.

Reply:
It used the fat jar when SDK is not given. It is a question weather we should support that.

Reply:
IMO, it would be better if we base on the SDK.

Reply:
It does use the langserver.jar if the sdk path is not set. https://github.com/ballerinalang/plugin-vscode/blob/0026d892ebbc97b34ba7cb16d298330ab2a1f90d/extension.js#L7